### PR TITLE
generate collection permalink

### PR DESCRIPTION
**Pull request by mkrajewski90**
PR to generate different permalinks by locales

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ strapi:
             # type: photos
             # Permalink used to generate the output files (eg. /articles/:id).
             permalink: /photos/:id/
+            # Permalinks defined for different locales
```
Comment (bluszcz) [inline README.md:42]: Do you think we could document it bit more? :) Otherwise I think it looks good.

**Review by bluszcz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mkrajewski90) [reply README.md:42]: I've added a paragraph to README

**Comment by bluszcz** [comment]
Thanks! Have a nice holidays!

**Comment by mkrajewski90** [comment]
Thanks! When will you merge develop brach to master?

**Comment by bluszcz** [comment]
30m ;)

**Pull request merged**